src/converter.ts

```diff
@@ -95,19 +84,14 @@ export class Warning extends Error {
 	description: string
 
 	constructor({
-		name,
-		message,
-		description,
+		nRFAssetTracker,
```

Comment:
nRFAssetTracker is a name of proyect.

Refer to TypeScript type there as the type. Use `keyof`